libs/mark-flow-ui/src/components/review.tsx

```diff
@@ -210,7 +204,7 @@ export function Review({
             footerAlign={selectionsAreEditable ? 'right' : undefined}
             footer={
               selectionsAreEditable && (
-                <DecoyButton>
+                <Button onPress={() => {}}>
```

Comment:
Based on your commit description, should you be passing a `tabIndex` here?

Reply:
Great catch, I think it got removed accidentally in a rebase